src/metabase/api/pulse.clj

```diff
@@ -111,9 +111,9 @@
   [id]
   (let [card (Card id)]
     (read-check Database (:database (:dataset_query card)))
-    (let [data (:data (qp/dataset-query (:dataset_query card) {:executed_by *current-user-id*}))
+    (let [result (qp/dataset-query (:dataset_query card) {:executed_by *current-user-id*})
           ba   (binding [render/*include-title* true]
-                 (render/render-pulse-card-to-png card data))]
+                 (render/render-pulse-card-to-png card result))]
```

Comment:
I think there's at least one other instance where we called `render-pulse-card-to-png`.